### PR TITLE
add pf-m-wrap to MD th elements

### PR DESCRIPTION
**Pull request by redallen**
Makes all table headers wrap per discussion with @andybraren 

cc @mcarrano

### CODE REVIEW

File: packages/theme-patternfly-org/scripts/md/styled-tags.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ function styledTags() {
               tr.children
                 .filter(child => child.tagName === 'th')
                 .forEach(th => {
+                  th.properties.className = th.properties.className || '';
+                  th.properties.className += ' pf-m-wrap';
```
Comment (andybraren) [inline packages/theme-patternfly-org/scripts/md/styled-tags.js:42]:
```suggestion
                  th.properties.classList.add('pf-m-wrap');
```

My JS game is weak, but would something like this work?

**Review by andybraren — APPROVED** [review]
Ah yep, fixed. The preceding space that can occur in the `class` attribute is a little ugly, but if you're cool with merging then I am.

```
<th class=" pf-m-wrap" role="columnheader" scope="col">
```

**Comment by mcarrano** [comment]
This is great @redallen !  So does this mean that tables written in Markdown will now wrap properly? No more need to code these directly in HTML?

cc @mmenestr

**Comment by andybraren** [comment]
@mcarrano They'll no longer be clipped and hopefully wrap better in general, at least. See some discussion & screenshots [here](https://patternfly.slack.com/archives/CB9KCDK4M/p1615997027005500?thread_ts=1615991480.000800&cid=CB9KCDK4M).

File: packages/theme-patternfly-org/scripts/md/styled-tags.js (hunk 1 above)
Comment (redallen) [reply packages/theme-patternfly-org/scripts/md/styled-tags.js:42]: Nope. This script is converting AST nodes from MD into JSX.

The leading space doesn't matter except for aesthetics so I'm okay with it.

**Pull request merged**